enumerable.rb

```diff
@@ -0,0 +1,85 @@
+module Enumerable
+    def my_each
+        for item in self
+            yield(item)
+        end
+        self
+    end
+   
+    def my_each_with_index
+        i = 0
+        while i < self.length
+         yield(self[i], i)
+        i += 1
+    end
```

Comment:
- [ ] returns the same thing as ruby's [`each_with_index`](https://ruby-doc.org/core-2.6.4/Enumerable.html#method-i-each_with_index). ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/advanced_building_blocks_enumerable/my_each_with_index.png) from [Odin](https://www.theodinproject.com/courses/ruby-programming/lessons/advanced-building-blocks#assignment-2)).
Returns an enumerator if no block is given. Can be implemented as above.